power_grid_model_c/power_grid_model/include/power_grid_model/component/node.hpp

```diff
@@ -57,13 +57,13 @@ class Node final : public Base {
         return get_sc_output(uabc_pu);
     }
     template <symmetry_tag sym> NodeOutput<sym> get_null_output() const {
-        NodeOutput<sym> output{};
+        NodeOutput<sym> output{.u_pu = {}, .u = {}, .u_angle = {}, .p = {}, .q = {}};
         static_cast<BaseOutput&>(output) = base_output(false);
         return output;
```

Comment:
it might be better to do something like
```cpp
auto output = base_output<NodeOutput<sym>>(false);
output.u_pu = {};
output.u = {};
output.u_angle = {};
output.p = {};
output.q = {};
```

Reply:
@mgovers can you make this change?

Reply:
the main reason for this comment was that i thought that the compiler might complain. since it doesn't, i'm OK with the current implementation.

whichever you prefer

Reply:
Please approve the PR.

Reply:
It's on merge queue